### DISCUSSION TITLE
Longhorn latest version compatibility with k8s and RHEL

### DISCUSSION THREAD

**Discussion by tusharpimple1983** [longhorn/longhorn#9425] [Q&A]
Hello All,

As per official longhorn version 1.7.1 best practices, it is tested upto k8s version 1.28 and rhel version 9.3

I am planning to use Longhorn latest version 1.7.1 with k8s version 1.29 and rhel version 9.4. Is this going to cause compatibility issues?

suggestions pls...

Regards,
Tushar Pimple

**Comment by derekbit** [comment]
The reason that we only tested RHEL 9.3 is RHEL 9.4 arm64 image was not published yet.
Of cause, you can test RHEL version 9.4.

**Reply by tusharpimple1983** [reply]
Thanks Derek for quick response, appreciate that...
